.github/workflows/release.yml

```diff
@@ -34,6 +34,8 @@ jobs:
         password: ${{ secrets.GITHUB_TOKEN }}
     - uses: docker/setup-qemu-action@v3
     - uses: docker/setup-buildx-action@v3
+      with:
+        platforms: linux/amd64,linux/arm64
```

Comment:
Do you also need to add a `platforms` argument to `build-push-action`? My assumption is that as-is you're just adding build support to the node and the build action needs to be instructed to build for those platforms.

Reply:
I just looked at https://docs.docker.com/build/ci/github-actions/multi-platform/ and my takeaway from that is that  you do need a `platforms` argument for the `build-push-action` step.

Reply:
Good catch!